source/js/GliderRecipe.js

```diff
@@ -0,0 +1,201 @@
+class GliderRecipe extends HTMLElement {
+  constructor(){
+    super();
+    let shadow = this.attachShadow({mode: 'open'});
```

Comment:
Codacy has a fix for the issue: ['shadow' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8437408)
```suggestion
    const shadow = this.attachShadow({mode: 'open'});
```